vividus-tests/src/main/resources/story/system/ImageBasedVisualStepsTests.story

```diff
@@ -8,3 +8,13 @@ When I compare_against baseline with name `context-element-with-acceptable-diff-
 When I compare_against baseline with name `context-element-with-acceptable-diff-percentage` from image `${response-as-bytes}` using storage `filesystem` and ignoring:
 |ACCEPTABLE_DIFF_PERCENTAGE|
 |100                       |
+
+Scenario: Verify visual checks using image out of file
+When I compare_against baseline with name `context` from image out of file `src/main/resources/baselines/context.png`
+When I compare_against baseline with name `context` from image out of file `src/main/resources/baselines/context.png` using storage `filesystem`
```

Comment:
The steps should work for the absolute path and for the 'resources' (/baselines/context.png))

Reply:
I'm ok with absolute path, but I don't agree for 'resources' as it can confuse users